crates/rooch/src/commands/init.rs

```diff
@@ -27,9 +28,9 @@ pub struct Init {
     pub server_url: Option<String>,
     #[clap(flatten)]
     pub context_options: WalletContextOptions,
-    /// Whether a non-empty password should be provided to rooch.keystore when it comes to the init command
-    #[clap(long = "encrypt-keystore")]
-    pub encrypt_keystore: Option<bool>,
+    /// This is a tweak command flag to skip the integration test when executing cmd.feature file or use with rooch init to ignore entering password
```

Comment:
The command help also needs to improve. The user does not know the `cmd.feature` things.

Reply:
@feliciss I will merge this PR first. You can improve it in the new PR.

Reply:
> > The command help also needs to improve. The user does not know the `cmd.feature` things.
> 
> This comment is not only for the users, but the developers. Thus, I don't think it's entirely for users. The users may also include developers.

Yes, the major users are the DApp developer on rooch, but not rooch's developer. DApp developers do not need to know what `cmd.feature` things.  If the comment is for rooch's developer, it can be the code comment, but not the command help.

Reply:
> > > The command help also needs to improve. The user does not know the `cmd.feature` things.
> > 
> > 
> > This comment is not only for the users, but the developers. Thus, I don't think it's entirely for users. The users may also include developers.
> 
> Yes, the major users are the DApp developer on rooch, but not rooch's developer. DApp developers do not need to know what `cmd.feature` things. If the comment is for rooch's developer, it can be the code comment, but not the command help.

Got it.